services/alerta/service.go

```diff
@@ -204,6 +215,13 @@ func (s *Service) preparePost(token, tokenPrefix, resource, event, environment,
 	if len(service) > 0 {
 		postData["service"] = service
 	}
+	postData["timeout"] = int64(timeout / toml.Duration(time.Second))
```

Comment:
Once the types are changed to `time.Duration` this can be simplified as `int64(timeout/time.Second)`